management/src/main/java/org/ehcache/management/registry/DefaultCollectorService.java

```diff
@@ -84,7 +85,9 @@ public synchronized void start(ServiceProvider serviceProvider) {
     cacheManager = serviceProvider.getService(CacheManagerProviderService.class).getCacheManager();
     scheduledExecutorService = serviceProvider.getService(ExecutionService.class).getScheduledExecutor(configuration.getCollectorExecutorAlias());
 
-    cacheManager.registerListener(this);
+    if (cacheManager instanceof ObservableCacheManager) {
```

Comment:
Hmmm ... looks like I dropped the `TODO` in one of my rebases.

This affects `DefaultCollectorService`, `DefaultSharedManagementService`, and `DefaultManagementRegistryService`.  Do we need the warning in each place?